kv/txn.go

```diff
@@ -26,14 +26,14 @@ import (
 )
 
 // RunInNewTxn will run the f in a new transaction environment.
-func RunInNewTxn(store Storage, retryable bool, f func(txn Transaction) error) error {
+func RunInNewTxn(store Storage, txnScope string, retryable bool, f func(txn Transaction) error) error {
```

Comment:
Could we add another function to allow using local txnScope so we don't have to pass in `GlobalTxnScope` everywhere?